src/cli.js

```diff
@@ -100,8 +116,15 @@ function Validator() {
       fs.readFileSync(filename, "utf8").toString(),
       path.extname(filename)
     );
+
+    for (const catalog of args.catalogs) {
+      if (!fs.existsSync(catalog)) {
+        throw new Error(`❌ Catalog not found: ${catalog}`);
+      }
+    }
```

Comment:
I think we can drop this check and make it 'lazy'. If we follow the "priority" model and the user passed two `--catalogs` and we found a match in the first, it doesn't matter the second was invalid. If we try to read one and it doesn't exist, it will throw anyway. We just throw `ENOENT: no such file or directory` if filename is invalid so (local) catalog can be the same.

Reply:
Fixed by dad46a136bc280a1863005a941aa9363e03e0a28